server/config/const.go

```diff
@@ -3,6 +3,8 @@
 
 package config
 
+// REVIEW: need an interface for returning bot info
+// probably good to have a struct to capture the data clump
 const (
 	BotUserName    = "gcal"
 	BotDisplayName = "Google Calendar"
```

Comment:
all fields being used here, that should be put into a struct

	BotUserName    = "gcal"
	BotDisplayName = "Google Calendar"
    BotDescription = "Created by the Google Calendar Plugin."
	ApplicationName    = "Google Calendar"
	Repository         = "mattermost-plugin-gcal"
	CommandTrigger     = "gcal"
	TelemetryShortName = "gcal"